portfolio/src/main/webapp/script.js

```diff
@@ -48,10 +40,8 @@ function showSlides(n) {
   }
 
   for (i = 0; i < slides.length; i++) {
-      slides[i].style.display = "none";
-  }
-  for (i = 0; i < dots.length; i++) {
-      dots[i].className = dots[i].className.replace(" active", "");
+    slides[i].style.display = "none";
```

Comment:
I am assuming "none", "active", "block",  and "" have important meaning. I would create an enum to store all these variables. What do you think?

Reply:
I agree, added an enum storing all the variables with comments describing what each "state" means